lib/puppet/util/package.rb

```diff
@@ -1,8 +1,13 @@
 module Puppet::Util::Package
   def versioncmp(version_a, version_b)
```

Comment:
Given this function has worked this way for so long, I'm wondering if the caller should pass in an argument to opt into the new behavior?